src/main/java/com/kodilla/ecommercee/OrderController.java

```diff
@@ -1,4 +1,43 @@
 package com.kodilla.ecommercee;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+@RestController
+@RequestMapping("/v1/order")
 public class OrderController {
+
+
+    @Autowired
+    private ServiceOrder service;
+
+    @Autowired
+    private OrderMapper orderMapper;
+
+    @GetMapping(value = "getOrders")
```

Comment:
Dostosuj jeszcze te mapowania do artykułu który podesłalem. Czyli to zostawiamy po prostu jako /v1/order a nie /v1/order/getOrders , i analogicznie create, update i delete